src/sbt-test/interplay/library-with-plugin/test

```diff
@@ -7,26 +7,30 @@ $ exec git add .
 $ exec git commit -m commit
 
 # Setup remote git repo in target directory
-$ exec git init --bare target/remote
-$ exec git remote add origin target/remote
+$ exec git init --bare $PWD/target/remote
```

Comment:
For some reason, after some git update, it was not working with relative paths anymore.